### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by 1e-to**
Fix examples

### CODE REVIEW

**Comment by 1e-to** [comment]
Do we need update lines in commands/simple_sum?

File: numba_dppy/examples/debug/simple_dppy_func.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2020, 2021 Intel Corporation
```
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/simple_dppy_func.py:1]: Why dppy_func.py is not simple enough?

File: numba_dppy/examples/debug/simple_sum.py [hunk 2]
```diff
@@ -16,7 +30,13 @@ def data_parallel_sum(a, b, c):
 b = np.array(np.random.random(N), dtype=np.float32)
 c = np.ones_like(a)
 
-with dpctl.device_context("opencl:gpu") as gpu_queue:
-    data_parallel_sum[global_size, dppy.DEFAULT_LOCAL_SIZE](a, b, c)
+# Use the environment variable SYCL_DEVICE_FILTER to change the default device.
+# See https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#sycl_device_filter.
+device = dpctl.select_default_device()
+print("Using device ...")
+device.print_device_info()
```
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/simple_sum.py:37]: It is not simple now.

**Review by PokhodenkoSA — CHANGES_REQUESTED** [review]
Please, see comments.

File: docs/user_guides/debugging/index.rst [hunk 3]
```diff
@@ -46,13 +46,13 @@ Running GDB and creating breakpoint in kernel:
 
     $ export NUMBA_DPPY_DEBUGINFO=1
     $ gdb-oneapi -q --args python simple_sum.py
-    (gdb) break simple_sum.py:8  ### Set breakpoint in kernel
+    (gdb) break simple_sum.py:22  ### Set breakpoint in kernel
     (gdb) run
     ...
-    Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at simple_sum.py:8
+    Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at simple_sum.py:22
     8           i = dppy.get_global_id(0)
```
Comment (PokhodenkoSA) [inline docs/user_guides/debugging/index.rst:53]: 22?

File: numba_dppy/examples/debug/simple_dppy_func.py (hunk 1 above)
Comment (1e-to) [reply numba_dppy/examples/debug/simple_dppy_func.py:1]: Removed unnecessary functions and comments. Now simple_dppy_func is rather simplified

File: docs/user_guides/debugging/index.rst (hunk 3 above)
Comment (1e-to) [reply docs/user_guides/debugging/index.rst:53]: Yes https://github.com/IntelPython/numba-dppy/pull/455/files#diff-2daedece066f78e47b74684e34cfdd69085f711cb837dc96510dbf4975e0934cR22

**Review by PokhodenkoSA — APPROVED** [review]

**Pull request merged**